src/_cffi_src/build_bindings.py

```diff
@@ -1,19 +1,14 @@
 import os
 import re
-import sys
 from cffi import FFI
 
 
-# see: https://docs.python.org/3/library/platform.html#platform.architecture
-is_64bits = sys.maxsize > 2 ** 32
-
-
 BASEDIR = os.path.dirname(os.path.abspath(__file__))
 
 # import `get_winfsp_dir` the violent way given winfspy cannot be loaded yet
-get_winfsp_dir = None
 exec(open(f"{BASEDIR}/../winfspy/plumbing/get_winfsp_dir.py").read())
```

Comment:
We should be able to replace this by:
```python
g = {}
exec(open(f"{BASEDIR}/../winfspy/plumbing/get_winfsp_dir.py").read(), g) 
WINFSP_DIR = g['get_winfsp_dir']()
```

Hence `get_winfsp_dir` wouldn't become a function out of thin air (and require the `# noqa# tag)

Reply:
:+1: